lib/generators/noticed/model_generator.rb

```diff
@@ -42,7 +42,7 @@ def model_path
       end
 
       def params_column
-        case ActiveRecord::Base.configurations.configs_for(spec_name: "primary").config["adapter"]
+        case ActiveRecord::Base.connection_config[:adapter]
```

Comment:
We may need to use this conditionally because it's being removed:

>DEPRECATION WARNING: connection_config is deprecated and will be removed from Rails 6.2 (Use connection_db_config instead) (called from <main> at (irb):1)

```
ActiveRecord::Base.connection_db_config.adapter
#=> "sqlite3"
```

I'm thinking this would be most reliable:

```ruby
adapter = ActiveRecord::Base.respond_to?(:connection_db_config) ? ActiveRecord::Base.connection_db_config.adapter : ActiveRecord::Base.connection_config[:adapter]
```

Reply:
Good call!
Have broken it out into an if / else instead of the ternary because of line length.

Will need the older connection_config[:adapter] version in-place until support ends for Rails 6.x.  (Unless we'd like to also polyfill that one, and I'm not opposed!)

Reply:
This will work just fine. 👍